packages/wallet/src/redux/protocols/funding/player-b/container.tsx

```diff
@@ -36,8 +38,11 @@ class FundingContainer extends PureComponent<Props> {
           />
         );
       case states.WAIT_FOR_FUNDING:
-        // TODO: embed the funding container
-        return <div />;
+        if (isTerminal(state.fundingState)) {
```

Comment:
This check is required since the compiler doesn't know that `state.fundingState` is never terminal. Should that ever occur?

Reply:
Updated